contracts/pool/src/tests/budget.rs

```diff
@@ -564,11 +575,27 @@ fn upgrade_debt_token() {
     let debt_token_v2_wasm = env.deployer().upload_contract_wasm(debt_token_v2::WASM);
     let asset = sut.reserves[0].token.address.clone();
 
-    measure_budget(&env, nameof(upgrade_debt_token), || {
+    measure_budget(&env, function_name!(), || {
         sut.pool.upgrade_debt_token(&asset, &debt_token_v2_wasm);
     });
 }
 
+#[test]
+fn s_token_transfer() {
+    let env = Env::default();
+    env.mock_all_auths();
+
+    let sut = init_pool(&env, true);
+    let (_, from_borrower, _) = fill_pool(&env, &sut, true);
```

Comment:
I suppose, 3 `deposit`s would be much heavier than 2 `deposit`s and 1 `borrow`, since the heaviest call in `calc_account_data` is in the `is_using_as_collateral` branch.

Reply:
We will not fall into calc_account_data if sender doesn't have debt

Reply:
Oh, I see.